server/users.js

```diff
@@ -373,7 +373,7 @@ class Connection {
 		this.autojoins = '';
 	}
 	/**
- 	* @param {string | BasicRoom?} roomid
+ 	* @param {string | number | BasicRoom?} roomid
```

Comment:
There is 800% no reason why `roomid` needs to be a number here.

Reply:
I think you're right and this method isn't actually called anywhere with a number as roomid, consistency is important however. The preferred option would obviously be to make ALL id's either numbers or string, not string | number, but that would require more code to be changed because they're used interchangeably in different places.

Reply:
`roomid`s have _always_ been strings, they look like this: `'lobby'` or `'help-guest123'` or `'battle-ou-12345'`. Are you getting them mixed up with `socketid`s or something?

Reply:
Possibly. As I said, only made this change for the sake of consistency. I can revert that back.